### DISCUSSION TITLE
Commitlint?

### DISCUSSION THREAD

**Discussion by dylanhitt** [go-fuego/fuego#153] [General]
Any thoughts on adding [commitlint](https://github.com/conventional-changelog/commitlint)? This would make the contributions look more uniform.

**Comment by EwenQuim** [comment]
I think this has little value and creates friction for contributors. 

But adding pre-push hooks that run `make ci` can be useful (to make sure the ci will be ok once pushed).

**Reply by dylanhitt** [reply]
as long as they're opt in I don't really mind. I personally don't like pre-commit hooks. I'll close the discussion.

**Reply by EwenQuim** [reply]
Thank you for the proposition.

And about pre commit hooks : they are opt-in by default since they require action on the local `.git` folder.

**Reply by dylanhitt** [reply]
eh, I find people try to hide the opt-in'ness them in a makefile or in npm nowadays.
